HW02_B_Mimic_starter.Rmd

```diff
@@ -103,9 +104,11 @@ Now using the above mpg dataset, make this graph
 
 ```{r graph 4 code}
 ggplot(mpg,mapping = aes(x = cty, y = class)) + 
-  geom_jitter(shape=16, position=position_jitter(0.2),aes(color=class))+
+  geom_jitter(shape=16, position=position_jitter(0),aes(color=class))+
```

Comment:
This graph looks very nice too! It looks like you may have forgotten to include the title, however. The gridlines and part of the panel border have been removed on the original graph, which could be done with these lines:   theme(panel.grid = element_blank()) + #remove grid
  theme(panel.border = element_blank()) + #remove border
  theme(axis.line = element_line(color = "black")) + #keep axes, in black